invoke.go

```diff
@@ -93,11 +93,16 @@ func InvokeRPC(ctx context.Context, source DescriptorSource, ch grpcdynamic.Chan
 	if svc == "" || mth == "" {
 		return fmt.Errorf("given method name %q is not in expected format: 'service/method' or 'service.method'", methodName)
 	}
+
 	dsc, err := source.FindSymbol(svc)
 	if err != nil {
 		if isNotFoundError(err) {
 			return fmt.Errorf("target server does not expose service %q", svc)
 		}
+		// return the error unstringified if it is a gRPC status.Status error
+		if _, ok := status.FromError(err); ok {
```

Comment:
I realize that this is not a backwards compatible change. I'm not sure how inject any conditional logic short of importing `"cmd/grpcurl".formatError` here, and I don't think that's acceptable.